templates/request.html

```diff
@@ -244,7 +244,11 @@
                 map.setView([lat, lon], 14);
                 //L.marker([lat, lon]).addTo(map);
                 L.geoJson(data).addTo(map)
-            }
+			}
+			
+			$('.open-map-button').hover(
+               function(){ $(this).addClass('hover') },
+            )
```

Comment:
@MeliAlbu jQuery se desaconseja, le queda poco tiempo de vida. Convertir a JS vanilla de la siguiente manera:

```suggestion
            document.querySelector(".open-map-button").addEventListener("hover", function() {
              this.classList.add("hover")
            });
```